Tools/ardupilotwaf/boards.py

```diff
@@ -217,10 +220,7 @@ def configure_env(self, cfg, env):
             CONFIG_HAL_BOARD_SUBTYPE = 'HAL_BOARD_SUBTYPE_NONE',
         )
 
-        if not cfg.env.DEBUG:
-            env.CXXFLAGS += [
-                '-O3',
-            ]
+        self.optimize = ['-O3']
```

Comment:
This should be in an `__init__` method, as it is for PX4 - putting it here won't do anything because the super configure_env is called before 🙂 Same applies for Linux.

Reply:
fixed, thanks